products/main.css

```diff
@@ -103,4 +103,144 @@
   width: 80%;
   font-size: 0.5rem;
   text-align: center;
+}
+
+/* for powder_lc.html */
+.item-details {
+  display: flex;
+  justify-content: center;
+  align-items: left;
+  flex-direction: column;
+  /* flex-wrap: wrap; */
+  padding: 0.8rem 0;
+  width: 80%;
+}
+
+.item-buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.cap-block {
+  margin-bottom: 1rem;
+}
+
+.normal-block::before {
+  content: ".";
+  position: absolute;
+  display: block;
+  pointer-events: none;
+  color: rgba(0,0,0,0);
+  width: 0;
+  height: 0;
+  border-style: solid;
+  /* top: 100%;
+  left: 50%; */
+  margin-left: -5px;
+  border-width: 8px 5px 0 5px;
+  border-color: #d0d0d0 transparent transparent transparent;
+}
+
+.product-name {
+  font-size: 1rem;
+  line-height: 1rem;
+}
+
+.product-sub {
+  font-size: 0.6rem;
+  line-height: 0.6rem;
+  font-weight: 0;
+  color: #949494;
+}
+
+.tagline {
+  margin: 2rem 0;
+}
+
+.cap-block {
+  border: 1px solid var(--body-font-color--black);
+  padding: 2rem;
+  padding-bottom: 0;
+  font-size: 0;
+  display: inline-block;
+}
+
+.cap-image {
+  width: 50px;
+}
+
+.cap-desc {
```

Comment:
code duplication?

Reply:
.cap-desc

Reply:
how about you use bem for css naming?

it shows structure of each element.
some don't like it because of long names, but i like it.

merit is bigger than demerit to me.

Reply:
Thank you for your response, I'll change it like what you say.